src/main/scala/org/broadinstitute/dsde/firecloud/webservice/CookieAuthedApiService.scala

```diff
@@ -1,34 +1,37 @@
 package org.broadinstitute.dsde.firecloud.webservice
 
+import akka.actor.Props
 import org.broadinstitute.dsde.firecloud.dataaccess.HttpGoogleServicesDAO
 import org.broadinstitute.dsde.firecloud.model.UserInfo
 import org.broadinstitute.dsde.firecloud.service._
 import org.slf4j.LoggerFactory
-import spray.http.OAuth2BearerToken
+import spray.http._
 import spray.routing._
 
+import scala.language.postfixOps
+
 /**
  * Created by dvoet on 11/16/16.
  */
-trait CookieAuthedApiService extends HttpService with PerRequestCreator with FireCloudDirectives
-  with FireCloudRequestBuilding {
+trait CookieAuthedApiService extends HttpService with FireCloudDirectives with FireCloudRequestBuilding {
 
   val exportEntitiesByTypeConstructor: UserInfo => ExportEntitiesByTypeActor
 
   private implicit val executionContext = actorRefFactory.dispatcher
   lazy val log = LoggerFactory.getLogger(getClass)
 
-  def cookieAuthedRoutes: Route =
+  val cookieAuthedRoutes: Route =
     // download "proxy" for TSV files
-    path("workspaces" / Segment / Segment/ "entities" / Segment/ "tsv") {
+    path( "cookie-authed" / "workspaces" / Segment / Segment/ "entities" / Segment / "tsv" ) {
```

Comment:
the `cookie-authed` prefix needs to be on both the TSV path and the GCS download path.